src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1044,6 +1045,8 @@ internal void OnMouseEnter()
     internal void OnMouseExit()
     {
         editor.ShowHover = selectedEditorTab == EditorTab.CellEditor;
+        editor.ShowHexGrid = selectedEditorTab == EditorTab.CellEditor &&
```

Comment:
I think the editor class should determine when to show the hex grid, which is basically when there is an organelle selected and the cell preview is not visible